hasura.planx.uk/metadata/tables.yaml

```diff
@@ -566,7 +647,7 @@
         computed_fields:
           - data_merged
         filter: {}
-        allow_aggregations: true
```

Comment:
@DafyddLlyr see this as a change, but don't remember changing it. Any idea what it is and how to change it back?

Reply:
Good spot.

It looks like this has been checked somewhere https://hasura.io/docs/2.0/auth/authorization/permissions/aggregation-permissions/

There aren't many places we use these types of query - I'd check what we do for another corresponding permission set.

Reply:
Found and resolved this! Thanks